source/simulator/assembly.cc

```diff
@@ -503,6 +586,7 @@ namespace aspect
            * any other variable outside the block we are currently considering)
            */
           std::vector<types::global_dof_index>   local_dof_indices;
+          std::vector<std::vector<types::global_dof_index> >   local_neighbor_dof_indices;
```

Comment:
Let's just call it `neighbor_dof_indices` (to me, "local" always refers to the "current cell") :-)